### PR TITLE
fixed broken links issue

### PR DESCRIPTION
**Pull request by Heun11**

### CODE REVIEW

**Comment by Heun11** [comment]
fixes #144

**Comment by K20shores** [comment]
hi @Heun11 thanks for this! This is close to what we want;

File: src/main.jsx [hunk 1]
```diff
@@ -24,6 +24,10 @@ let routes = [
     path: "/",
     element: <Home />,
   },
+  {
+    path: "/home",
+    element: <Home />
```
Comment (K20shores) [inline src/main.jsx:29]:
```suggestion
    element:  <RedirectToRoot />
```

Hi @Heun11, we are actually looking for a redirect. A component that simply navigates to `/` would do

```
import { createBrowserRouter, RouterProvider, Navigate } from "react-router-dom";

const RedirectToRoot = () => <Navigate to="/" />;
```

Would you mind making that change?

**Review by K20shores — CHANGES_REQUESTED** [review]

**Review by K20shores — APPROVED** [review]

File: src/main.jsx (hunk 1 above)
Comment (Heun11) [reply src/main.jsx:29]: Sure, I'll make new PR

**Pull request merged**